members/ahmedHamdy.html

```diff
@@ -0,0 +1,153 @@
+											<a href="https://github.com/AhmedHHamdy">Github</a>
+										</li>
+										<li>
+											<h4 class="icon solid fa-brands fa-twitch"><span class="label">Twitch</span></h4>
+											<a href="https://www.twitch.tv/ahmedhcodes">Twitch</a>
+										</li>
+									</ul>
+								</section>
+						</div>
+					</div>
+				</div>
+			</section>
+
+		<!-- Footer -->
+		<div id="footer">
+			<div class="container">
+				<div class="row gtr-200 gtr-uniform">
+					<section class="col-4 col-12-medium col-12-small">
+						<header class="major">
+							<h3>About This Page:</h3>
+						</header>
+						<p>All members are added voluntarily by each individual themselves on Github:</p>
+						<ul class="actions">
+							<li><a href="https://github.com/HattieTavares/100devs-Directory" class="button alt">Github Repo</a></li>
+						</ul>
+					</section>
+					<section class="col-4 col-6-medium col-12-small">
+						<header class="major">
+							<h3>Disclaimer</h3>
+						</header>
+						<ul class="updates">
+							<li>
+								<p>
+									This web site is not endorsed by, directly affiliated with, maintained, authorized, or sponsored by 100devs. All images are the sole property of their original owners.
+								</p>
+							</li>
+						</ul>
+					</section>
+					<section class="col-4 col-6-medium col-12-small">
+						<header class="major">
+							<h3>Learn with Leon</h3>
+						</header>
+						<ul class="labeled-icons">
+							<li>
+								<h4 class="icon solid fa-solid fa-house"><span class="label">100devs Home</span></h4>
+								<a href="https://leonnoel.com/100devs/">100devs Home</a>
+							</li>
+							<li>
+								<h4 class="icon solid fa-brands fa-twitch"><span class="label">Twitch</span></h4>
+								<a href="https://www.twitch.tv/learnwithleon">Learn with Leon-Twitch</a>
+							</li>
+							<li>
+								<h4 class="icon solid fa-brands fa-discord"><span class="label">Discord</span></h4>
+								<a href="https://leonnoel.com/discord">100devs Discord</a>
+							</li>
+							<li>
+								<h4 class="icon brands fa-brands fa-twitter"><span class="label">Twitter</span></h4>
+								<a href="https://twitter.com/leonnoel">@Leonnoel</a>
+							</li>
+							<li>
+								<h4 class="icon brands fa-brands fa-youtube"><span class="label">Youtube</span></h4>
+								<a href="https://www.youtube.com/channel/UCGiRSHBdWuCgjgmPPz_13xw">Learn with Leon-Youtube</a>
+							</li>
+						</ul>
+					</section>
+				</div>
+			</div>
+			<div class="copyright">
+				&copy; 2022
+			</div>
+		</div>
+
+		<!-- Scripts -->
+			<script src="assets/js/jquery.min.js"></script>
+			<script src="assets/js/jquery.dropotron.min.js"></script>
+			<script src="assets/js/jquery.poptrox.min.js"></script>
+			<script src="assets/js/browser.min.js"></script>
+			<script src="assets/js/breakpoints.min.js"></script>
+			<script src="assets/js/util.js"></script>
+			<script src="assets/js/main.js"></script>
+
+	</body>
+</html>
```

Comment:
See that red circle? That means no newline at the end of the document. It's considered best practice to have a new line.

Reply:
Oh ok, I never noticed it haha, I'll have to see if there's a way to fix it for all files.